xcm-simulator/src/storagehub/configs/mod.rs

```diff
@@ -959,3 +961,55 @@ impl pallet_bucket_nfts::Config for Runtime {
     #[cfg(feature = "runtime-benchmarks")]
     type Helper = ();
 }
+
+/****** Commit-Reveal Randomness pallet ******/
```

Comment:
That's the thing, I saw that in some config files we have pallets and their required structures and implementations all mixed together and it's kinda confusing because it's all over the place, so I wanted to set a precedent on how I'd like to have the pallets separated 😁

Reply:
I don't mind it, I think it's nice, but I'd implement it for the others too in that case.

Reply:
I just implemented this format for all pallets for the runtime! Let me know what you think and we can merge this one